Assets/_BForBoss/_Core/Scripts/Debug/DebugWindow.cs

```diff
@@ -45,9 +44,6 @@ private void Awake()
                 }
             }
             
-            var movement = FindObjectOfType<PlayerMovementBehaviour>();
-            var input = new Perigon.Character.InputSettings(movement.GetCharacterLook(), movement.actions);
-            _freezeActionsUtility = new FreezeActionsUtility(input);
```

Comment:
Don't need to freeze anything anymore, since it's handled in the new `PGInputSystem` on `BaseWorldManager`, the ActionMap get's swapped over to the UI mapping, and debug state takes advantage of that.